.github/workflows/esy-build.yml

```diff
@@ -35,3 +35,42 @@ jobs:
       - name: Run tests.
         if: ${{ matrix.system != 'windows' }}
         run: esy test
+
+      - name: Clone basic template
+        if: ${{ matrix.system == 'ubuntu' }}
+        run: |
+          cd ocaml-tree && npm install && cd ..
+          git clone -b directory-targets https://github.com/jchavarri/melange-basic-template.git
+
+      - name: Remove esy.lock in template
+        if: ${{ matrix.system == 'ubuntu' }}
+        run: |
+          cd ocaml-tree
```

Comment:
this should probably have been `melange-basic-template` @jchavarri

Reply:
Right. Fixed in #369.